mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.erb

```diff
@@ -2614,6 +2614,7 @@ resource "google_container_node_pool" "with_kubelet_config" {
       "https://www.googleapis.com/auth/logging.write",
       "https://www.googleapis.com/auth/monitoring",
     ]
+    logging_variant = "DEFAULT"
```

Comment:
I assume we have a bunch of tests where this is not set, so we don't need to add one?

Reply:
Yes, the majority of tests do not have this set